gradle/verification-metadata.xml

```diff
@@ -242,6 +268,101 @@
             <sha256 value="6d7c68423115f921718d944f859924b4c685217ec03a49f70455a8b2caa972e6" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.hamcrest" name="hamcrest-core" version="1.3">
+         <artifact name="hamcrest-core-1.3.jar">
+            <sha256 value="66fdef91e9739348df7a096aa384a5685f4e875584cce89386a7a47251c4d8e9" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="hamcrest-core-1.3.pom">
+            <sha256 value="fde386a7905173a1b103de6ab820727584b50d0e32282e2797787c20a64ffa93" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
+      <component group="org.hamcrest" name="hamcrest-parent" version="1.3">
+         <artifact name="hamcrest-parent-1.3.pom">
+            <sha256 value="6d535f94efb663bdb682c9f27a50335394688009642ba7a9677504bc1be4129b" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
```

Comment:
Is `hamcrest` actually required in order to support JUnit 5? If not, this should be reverted. If so, that's another third-party dependency we would need to get approval for.

Reply:
I will try block this and get back to you.

Reply:
Hamcrest could be blocked, tests still run.

Reply:
As it turns out, `hamcrest` does not need to be blocked, so you can restore it if it is easier. I'll leave it up to you.